k-distribution/include/builtin/kast.k

```diff
@@ -63,32 +63,39 @@ module KSEQ
 endmodule
 
 
-// To be used when parsing/pretty-printing symbolic configurations
-module KSEQ-SYMBOLIC
-  imports KSEQ
+module ML-SYNTAX
+  imports BASIC-K
+
   syntax #KVariable ::= r"(?<![A-Za-z0-9_\\$!\\?])(\\!|\\?)?([A-Z][A-Za-z0-9'_]*|_)"   [token, prec(1), autoReject]
-                      | #UpperId                                                       [token, autoReject]
-  syntax KConfigVar ::= r"(?<![A-Za-z0-9_\\$!\\?])(\\$)([A-Z][A-Za-z0-9'_]*)"          [token, autoReject]
-  syntax KBott      ::= #KVariable
-  syntax KBott      ::= KConfigVar
-  syntax KLabel     ::= #KVariable
 
-  syntax KBott ::= "#True" [klabel(#True), symbol, poly(0)]
-  syntax KBott ::= "#False" [klabel(#False), symbol, poly(0)]
-  syntax KBott ::= "#Not" "(" K ")" [klabel(#Not), symbol, mlOp, poly(0, 1), mlUnary]
-                 | "#Ceil" "(" K ")" [klabel(#Ceil), symbol, mlOp, poly(0; 1), mlUnary]
-                 | "#Floor" "(" K ")" [klabel(#Floor), symbol, mlOp, poly(0; 1), mlUnary]
+  syntax K ::= "#True" [klabel(#True), mlOp, symbol, poly(0)]
```

Comment:
Now that these are all syntax k again, you can replace the > blocks and remove the syntax priority.